tests/acceptance/ember-shepherd-test.js

```diff
@@ -250,49 +185,39 @@ module('Acceptance | Tour functionality tests', function(hooks) {
   test('Defaults applied', async function(assert) {
     assert.expect(1);
 
-    const defaults = {
-      classes: 'shepherd-theme-arrows test-defaults',
-      scrollTo: false,
-      showCancelLink: true
-    };
-
-    const steps = [{
-      id: 'test-defaults-classes',
-      options: {
-        attachTo: '.first-element bottom',
-        builtInButtons: [
-          {
-            classes: 'shepherd-button-secondary cancel-button',
-            text: 'Exit',
-            type: 'cancel'
-          },
-          {
-            classes: 'shepherd-button-primary next-button',
-            text: 'Next',
-            type: 'next'
-          }
-        ],
-        copyStyles: false,
-        highlightClass: 'highlight',
-        title: 'Welcome to Ember-Shepherd!',
-        text: ['Testing defaults']
+    const stepsWithoutClasses = [
+      {
+        id: 'test-highlight',
+        options: {
+          attachTo: '.first-element bottom',
+          builtInButtons: [
```

Comment:
Shouldn't we use `builtInButtons.cancel` etc here?

Reply:
Yep -- missed a few spots.

Reply:
Updated